### PR TITLE
Add top-navbar with useful links.

### PR DESCRIPTION
**Pull request by grayside**
Fixes https://github.com/phase2/rig/issues/126

I'm not sure I like the way the header looks, but the navbar seems nice. Incidentally adds more JS and upgrades to Bootstrap 4.

outrigger_dashboard

### CODE REVIEW

File: frontend/src/components/App.vue [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item"><a class="nav-link" href="http://outrigger.sh">What is Outrigger?</a></li>
+          <li class="nav-item"><a class="nav-link" href="http://docs.outrigger.sh">Documentation</a></li>
+          <li class="nav-item"><a class="nav-link" href="http://slack.outrigger.sh/">Join us on Slack</a></li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="supportDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              File an Issue
```
Comment (Jwaxo) [inline frontend/src/components/App.vue]: A++++ would file an issue again.

**Review by Jwaxo — APPROVED** [review]

**Comment by grayside** [comment]
Okay, made a few more tweaks. Here's what I got now:

### Desktop (Collapsed issues dropdown)

outrigger_dashboard_and_add_top-navbar_with_useful_links__by_grayside_ _pull_request__9_ _phase2_outrigger-dashboard_and_5__zsh

### Mobile (Expanded issues dropdown)

outrigger_dashboard

**Comment by grayside** [comment]
Now that I got #8 working, I found this breaks the dashboard, tables below the top DNS Records are all inline and squished.

File: frontend/src/components/Project.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="col project table-responsive">
+  <div class="project table-responsive">
```
Comment (Jwaxo) [inline frontend/src/components/Project.vue:2]: So did this clear up the "squishing" issue?

Comment (grayside) [reply frontend/src/components/Project.vue:2]: This part fixed each project's table being on the same horizontal row. Looks good on chrome at least.. I guess I should check others.

File: frontend/src/components/App.vue [hunk 3]
```diff
@@ -1,21 +1,44 @@
...
+            </div>
+          </li>
+        </ul>
       </div>
-    </div>
-    <div class="container">
+    </nav>
+
+    <header class="jumbotron">
+      <h1 class="display-3 mt-5">Outrigger Dashboard</h1>
```
Comment (grayside) [inline frontend/src/components/App.vue:33]: `mt-5` stands for `margin top 5`, where 5 is multiplied by some kind of predefined spacer and unit of measurement. display-3 addresses some sizing issues after I pulled out the extra markup.

**Pull request merged**